openapi/src/json-schemas/models/definitions/owner-id.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
-    "title": "Owner Id",
+    "title": "On-behalf-of",
```

Comment:
What's the reason behind this title change? Is it to make it clear that this owner ID directly maps to the on-behalf-of header? Is it not more correct to be called `Owner Id` as it already is? Just a thought

Reply:
Yep, I was confusing this with the header title. Have changed it back.